samples/snippets/src/main/java/com/example/monitoring/AlertPolicyEnableDisable.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.monitoring;
+
+// [START monitoring_enable_disable_alert_policy]
+
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.monitoring.v3.AlertPolicyServiceClient;
+import com.google.monitoring.v3.Aggregation;
+import com.google.monitoring.v3.AlertPolicy;
+import com.google.monitoring.v3.ComparisonType;
+import com.google.monitoring.v3.UpdateAlertPolicyRequest;
+import com.google.protobuf.BoolValue;
+import com.google.protobuf.Duration;
+import java.io.IOException;
+
+public class AlertPolicyEnableDisable {
+
+  private static final String METRIC_FILTER =
+          "metric.type=compute.googleapis.com/instance/"
+                  + "\"cpu/utilization\" AND resource.type=\"gce_instance\"";
+  private static final Duration AGGREGATION_DURATION = Duration.newBuilder().setSeconds(60).build();
+  private static final Aggregation AGGREGATION =
+          Aggregation.newBuilder()
+                  .setAlignmentPeriod(AGGREGATION_DURATION)
+                  .setCrossSeriesReducer(Aggregation.Reducer.REDUCE_MEAN)
+                  .setPerSeriesAligner(Aggregation.Aligner.ALIGN_MAX)
+                  .build();
+  private static final AlertPolicy.Condition.MetricThreshold METRIC_THRESHOLD =
+          AlertPolicy.Condition.MetricThreshold.newBuilder()
+                  .setComparison(ComparisonType.COMPARISON_GT)
+                  .addAggregations(AGGREGATION)
+                  .setFilter(METRIC_FILTER)
+                  .setDuration(AGGREGATION_DURATION)
+                  .build();
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String displayName = "alert-policy-display-name";
+    String alertPolicyId = "alert-policy-id";
+    // i.e projects/[PROJECT_ID_OR_NUMBER]/alertPolicies/[ALERT_POLICY_ID]
+    enableDisableAlertPolicy(alertPolicyId, false, displayName);
+  }
+
+  public static void enableDisableAlertPolicy(
+          String alertPolicyId, boolean status, String displayName) throws IOException {
```

Comment:
instead of passing individual fields it would be great if you pass an objects as parameter

i.e. `public static void enableDisableAlertPolicy(AlertPolicy.Condition condition,  AlertPolicy alertPolicy)`

Reply:
Here i am build condition object inside local method b'z i have do changes to use instance variable inside method as a local variable so i think it's not require to build condition instance inside static main method.